src/session_impl.cpp

```diff
@@ -1715,6 +1715,7 @@ namespace aux {
 		}
 
 		ret.udp_sock->set_force_proxy(m_settings.get_bool(settings_pack::force_proxy));
+		ret.udp_sock->set_proxy_settings(proxy());
```

Comment:
would you mind adding a comment explaining that this has to be called for the ``force_proxy`` settings to take effect? (assuming my understanding is correct). perhaps add more details too if you have a deeper understanding